lib/core/include/irods/irods_logger.hpp

```diff
@@ -484,14 +485,49 @@ namespace irods::experimental::log
             ///
             /// \since 4.3.0
             template <typename... Args>
-            auto operator()(fmt::format_string<Args...> _format, Args&&... _args) const -> void
+            void operator()(fmt::format_string<Args...> _format, Args&&... _args) const
```

Comment:
Let's use the trailing return type form for consistency.

Reply:
I'm not sure how that is done with a constraint.

Reply:
Ohhh interesting. I experienced a similar thing when attempting to use `override`.

Will look into that. cppreference must state something about it. Perhaps the wording is difficult to find.

Reply:
I figured it out.